figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -123,22 +144,21 @@ for (dataset in c("tcga", "pbta")) {
   dimnames(detect_mat)[[1]] <- all_caller_df$index
 
   # Turn into detected or not
-  detect_mat <- !is.na(detect_mat)
-
+  detect_mat_logical <- !is.na(detect_mat)
```

Comment:
If you follow my renaming suggestion
```suggestion
  detect_mat_logical <- !is.na(vaf_mat)
```